src/cron.ts

```diff
@@ -183,18 +200,26 @@ export const cron = async (startDate: Date, owner: string, repo: string, octokit
       }
 
       // if we don't find a match, we search for a closed referenda
-      const completedMatch = completedReferendas.find(
+      const completedMatch = finishedReferendas.find(
         ({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex(),
       );
       if (completedMatch) {
-        logger.info(`Found completed referenda for PR #${pr}`);
+        logger.info(`Found finished referenda for PR #${pr} with state ${completedMatch.state}`);
         const command = `/rfc process ${completedMatch.executedHash}`;
-        const msg = "PR can be merged. Write the following command to trigger the bot" + `\n\n\`${command}\``;
+        const rejectedMsg = `Referenda voting has finished with status \`${completedMatch.state}\``;
+        const finishedMsg =
+          "PR can be merged." + "\n\nWrite the following command to trigger the bot\n\n" + `\`${command}\``;
```

Comment:
The const names that remained are now a little inaccurate.

`finishedMsg` is used not in every finished state.
`rejectedMsg` is used in more than a rejected state.

Would those be more accurate?
`nonApprovedMsg` and `approvedMsg`, or something like that.